apps/site/src/_pages/docs/1_developing-blocks.mdx

```diff
@@ -259,11 +259,140 @@ whether they are connected to the block entity at all, and far away they are (wh
 
 There are messages for exploring the data available in the embedding application:
 
-- `queryEntities` allows you submit a query which filters the entities in the data store.
+#### Retrieving the subgraph rooted at a specific entity
 
-You can browse the available entities in order to display them, or create links between them.
+If you know of a specific entity that you want to retrieve more detailed information on, you can call `getEntity` with its `entityId`.
+When doing this, you can provide how much of the graph you want to explore around the entity, by providing `graphResolveDepths`.
 
-If you know of a specific entity that you want to retrieve more detailed information on, you can call `getEntity`.
+For example, to retrieve the subgraph rooted at the entity, with its links and their destinations:
+
+```js
+const getEntityResponse = await graphModule.getEntity({
+  data: {
+    entityId: someEntityId,
+    graphResolveDepths: {
+      hasLeftEntity: {
+        incoming: 1,
+        outgoing: 0,
+      },
+      hasRightEntity: {
+        incoming: 0,
+        outgoing: 1,
+      },
+    },
+  },
+});
+
+if (getEntityResponse.errors) {
+  // handle errors
+}
+
+const { data: entitySubgraph } = getEntityResponse;
+```
+
+Here `graphResolveDepths` specifies two edge kinds,
+
+- `hasLeftEntity` - the edge that appears between a link entity and its left entity
+- `hasRightEntity` - the edge that appears between a link entity and its right entity
+
+For each edge kind, we specify the `incoming` and `outgoing` depths to resolve, where:
```

Comment:
I got worried that this gets pretty heavy for an explanation on 'how do I get stuff linked from my block entity', and is the first time that the word 'traversal' appears in this document, so I attempted to explain it.

But I may have made it worse😭 
```suggestion
The process of exploring a graph is known as 'traversal'. This involves visiting a vertex (an element in the graph – in our case, an entity), and then following edges from it (connections to other vertices) based on the parameters of the query. The parameters specify which kinds of edges to follow, and how many times they should be followed for any given journey through the graph. Each vertex visited might have multiple edges that can be followed from it , which means that each query actually involves many different journeys through the graph, which we refer to as 'branches' of the traversal.

For each edge kind, we specify the `incoming` and `outgoing` depths to resolve, where:
```

Reply:
It's a tough one, I was trying to avoid blowing it up too much too